src/landpatterns/BGA/package.stanza

```diff
@@ -115,32 +92,58 @@ public defn BGA (
   lead-diam:Double,
   pitch:Double|Dims,
   package-body:PackageBody,
-  pad-planner:BGA-PadPlanner = Full-Matrix-Planner(),
+  pad-planner:BGA-PadPlanner = Full-Matrix-Planner(
+    rows = rows,
+    columns = columns,
+    pitch = pitch
+    ),
   lead-numbering:Numbering = Grid-Numbering(rows, columns)
   density-level:DensityLevel = DENSITY-LEVEL
 ) -> BGA :
   #BGA(
```

Comment:
For this first constructor, I wonder if we should just omit `pad-planner` and `lead-numbering` as arguments and just hardcode the defaults. `rows` and `columns` only have any use with the defaults and to me it would help make the difference between the constructors more apparent.

```
public defn BGA (
  --
  num-leads:Int,
  rows:Int,
  columns:Int,
  lead-diam:Double,
  pitch:Double|Dims,
  package-body:PackageBody,
  density-level:DensityLevel = DENSITY-LEVEL
) 
...
public defn BGA (
  --
  num-leads:Int,
  lead-diam:Double,
  package-body:PackageBody,
  pad-planner:BGA-PadPlanner,
  lead-numbering:Numbering = Grid-Numbering(rows(pad-planner), columns(pad-planner))
  density-level:DensityLevel = DENSITY-LEVEL
) -> BGA
```

Reply:
That's fair. I think I was primarily trying to preserve the existing interface so as not to break any existing code. 
But big picture - we're breaking a lot of interfaces already so its not a big deal.